### PR TITLE
Added thumbnail images to tutorials gallery

### LINKED ISSUE #944: Add thumbnail images to notebooks in documentation
### What is the expected enhancement?

The galleries, produced from the tutorial & migration notebooks, in the published documentation are all with the default image. It would be nice to have some better visuals where applicable - for instance [choosing a cell with some visual output and tagging it](https://nbsphinx.readthedocs.io/en/0.6.0/gallery/cell-tag.html) with `nbsphinx-thumbnail`

### PR DESCRIPTION
**Pull request by javabster**
### Summary

fixes #944 

I wasn't able to add images for all migration guides and tutorials (only ones which had `.png` files linked to them or in their output are able to be attached as thumbnail images.

### Screenshots
Screenshot 2022-12-13 at 4 39 17 PM
Screenshot 2022-12-13 at 4 31 39 PM

### CODE REVIEW

File: docs/tutorials/12_deuteron_binding_energy.ipynb [hunk 1]
```diff
@@ -1097,7 +1100,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.9.0 ('qiskit39')",
```
Comment (mrossinek) [inline docs/tutorials/12_deuteron_binding_energy.ipynb:1103]:
```suggestion
   "display_name": "Python 3",
```

File: docs/conf.py [hunk 2]
```diff
@@ -115,7 +115,16 @@
...
+nbsphinx_thumbnails = {
+    'migration/00_Migration_Guide_to_v0.5':
+    '_images/qiskit_nature_0.5_overview.png',
+    'tutorials/03_ground_state_solvers':
+    '_images/h2_gs.png',
+    'tutorials/04_excited_states_solvers':
+    '_images/h2_es.png',
+    'tutorials/06_qubit_mappers':
+    '_images/jw_mapping.png',
+}
```
Comment (mrossinek) [inline docs/conf.py:127]: To fix CI, all you need to do is run `make black` :+1:

**Review by mrossinek — COMMENTED** [review]
Thanks a lot, Abby! I guess in the future we can look for some additional thumbnails. For example for the electronic structure case this should be fairly simple to do by just picking one of the used molecules (I believe its LiH) and getting an image of that.
For the vibrational structure tutorial our example is always the vibrational data obtained for CO2. So one could look for a depiction of the vibrational modes of that molecule (for example [this one](https://www.researchgate.net/profile/S_Civis/publication/334276090/figure/download/fig1/AS:799358937202689@1567593430666/Vibrational-modes-of-CO-2-Vibrational-modes-where-the-symmetry-of-the-excited-state-is.png).

**Comment by javabster** [comment]
Added a few more images:
Screenshot 2023-01-06 at 11 23 54 AM
Screenshot 2023-01-06 at 11 24 03 AM

**Comment by mrossinek** [comment]
> Added a few more images

Thanks a lot! Maybe some suggestions:
- could you use the attached for electronic structure on both pages instead of the "scary-looking equations"?
- do we have some schematic depiction of what runtime does rather than having some "random results" as the thumbnail for that tutorial?

pyridine_lumo

**Comment by javabster** [comment]
> do we have some schematic depiction of what runtime does rather than having some "random results" as the thumbnail for that tutorial?

I could add this image from the runtime docs?
runtime-architecture

**Comment by javabster** [comment]
updated images:

Screenshot 2023-01-11 at 3 43 34 PM
Screenshot 2023-01-11 at 3 43 41 PM

**Review by mrossinek — APPROVED** [review]
Thanks a lot Abby! :+1:

**Review by mrossinek — APPROVED** [review]

**Pull request merged**